cybered/hashing/templates/hashing/tools.html

```diff
@@ -0,0 +1,86 @@
+<!doctype HTML>
+<html>
+{% include "includes/head.html" %}
+
+<body>
+    {% include "includes/header.html" %}
+    <div class="container">
+        <h3>Tools</h3>
+
+        <p>
+            Enter a message and press the 'Hash' button to see the message digests for the message.
+            If you enter a password, then you will also see the keyed digests.
+        </p>
+
+        {% if input_text %}
+        <div class="card">
```

Comment:
For this card and the others, I'm tempted to make the input, key, and combined input `<pre>`formatted text, but we might have to do some extra work to make line wrapping occur.

Reply:
It was doing some weird stuff with vertical spacing when I had them as `<pre>`, which is why I changed it. It's all text-only, so I felt like there's not a ton of reason to do it.